web/src/upload.js

```diff
@@ -437,16 +446,35 @@ export function setup_upload(config) {
     return uppy;
 }
 
-export function deactivate_upload(uppy, config) {
+export function deactivate_upload(config, uppy) {
```

Comment:
These parameters don't seem to match what the caller is using.

Reply:
I have over here, tried to keep the Uppy instance as an optional parameter that may be passed, if we want a specific use case sometime in the future. The call we make over at `message_edit.js` is `upload.deactivate_upload({mode: "edit", row: row_id});`, where `{mode: "edit", row: row_id}` corresponds to the `config` part of the parameter, and we are skipping the Uppy Instance param (undefined).

Reply:
Oh, that's confusing, but mainly because `config` is a weird name; maybe if you call that `opts` it'd be better? But I also don't see any purpose in supporting passing an `uppy` parameter, so would recommend not having that.

Reply:
Nah let's leave it as-is.